ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -126,6 +126,9 @@ define([
             }
             return value;
         },
+        transformValue: function (value, defaultValue, comparator) {
```

Comment:
Do we need 3 params in this function if you only ever use `value`?

Reply:
Removed, this was here on accident